tools/jdk/local_java_repository.bzl

```diff
@@ -60,6 +60,7 @@ def local_java_runtime(name, java_home, version, runtime_name = None):
         native.java_runtime(
             name = runtime_name,
             java_home = java_home,
+            visibility = ["//visibility:public"],
```

Comment:
Perhaps it would be better to add a visibility parameter with default value public. So that it can be changed if needed.

Reply:
@comius updated